.doctrine-project.json

```
@@ -5,17 +5,23 @@
     "slug": "mongodb-odm",
     "docsSlug": "doctrine-mongodb-odm",
     "versions": [
+        {
+            "name": "2.14",
+            "branchName": "2.14.x",
+            "slug": "2.14",
```

```suggestion
            "slug": "latest",
```

To become consistent with the other projects, latest has to be the newest upcoming version.